_includes/manuals/nightly/4.1.4_auditing.md

```diff
@@ -48,7 +51,9 @@ Example:
 
 1. Anonymizes all audits older then 7 days
 
-		<pre>foreman-rake audits:anonymize days=7</pre>
+```
+foreman-rake audits:anonymize days=7
+```
```

Comment:
This rendered incorrectly and it might be actually semantically incorrect, I spent some time fixing this in the script but then I realized it is just three occurrences so this will do it.

Reply:
IMHO that's already a good improvement. If you submit it as a separate PR I'll happily merge it already

Reply:
I had a look and https://github.com/theforeman/theforeman.org/pull/1918 does this and more.